### DISCUSSION TITLE
How to add google adsense correctly to monetize my blog in this beautiful theme?

### DISCUSSION THREAD

**Discussion by ekaingenieria** [adityatelange/hugo-PaperMod#422] [Q&A]
I would like to monetize my blog, but I don't know what is the correct way to do it. Has anyone been able to do it successfully?

**Comment by glenn-lover** [comment]
just try your best don't give up

**Comment by mustansir78** [comment]
I tried following this article: https://www.godo.dev/tutorials/hugo-in-article-ad/ but the snippet does not get rendered in the HTML. I really love this theme and wish there is an easier way via to config to integrate adsense into my blog.

@adityatelange I like your theme very much. I hope you can provide some inputs on how I can achieve this.
